base/src/jsMain/kotlin/KalugaThread.kt

```diff
@@ -23,10 +23,17 @@ actual class KalugaThread {
 
     actual companion object {
         actual val currentThread: KalugaThread get() = KalugaThread()
+        actual val MIN_PRIORITY: Int = 1
+        actual val MAX_PRIORITY: Int = 10
```

Comment:
as long as this is flexible per platform should this not be 1 (and of course the priority itself always be 1).